### PR TITLE
feat(ci): auto update for optimum-rbln

### PR DESCRIPTION
**Pull request by rebel-seinpark**
### 🚀 Summary of Changes

> *What does this PR do? What feature, fix, or improvement does it bring?*

---

### 📌 Related Issues / Tickets

* Resolves # https://github.com/rebellions-sw/vllm-rbln/issues/219
* Related to #

---

### ✅ Type of Change

* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`bug-fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [x] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [x] PR title follows Conventional Commits format
* [x] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: .github/workflows/dispatch_pkg_update.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            - "3.12"
+            - "3.13"
+        optimum-version:
+            description: Version of optimum-rbln to add
+            default: "latest"
+            type: string  
+        force-rebuild:
+            description: Whether to force-rebuild airflow image
+            default: false
+            type: boolean
```
Comment (rebel-sunwook) [inline .github/workflows/dispatch_pkg_update.yaml]: remove?

File: .github/workflows/pkg_update.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    SKIP_PR="false"
+                    fi
+                echo $CURRENT_VERSION
+                echo $SKIP_PR
+                echo "current-version=$CURRENT_VERSION" >> $GITHUB_OUTPUT
+                echo "skip=$SKIP_PR" >> $GITHUB_OUTPUT
+
+        outputs:
+            skip-airflow: ${{ steps.skip-airflow.outputs.skip }}
+            prod-image-tag: ${{ steps.skip-airflow.outputs.var }}
```
Comment (rebel-sunwook) [inline .github/workflows/pkg_update.yaml]: remove?

**Review by rebel-sunwook — APPROVED** [review]
👍
